### DISCUSSION TITLE
Типизация ymaps

### DISCUSSION THREAD

**Discussion by unn3daby** [yandex-maps-unofficial/vue-yandex-maps#64] [Q&A (1.x)] [answered]
Как типизировать ymaps в скрипте?

**Comment by daniluk4000** [comment, marked as answer]
Вы можете установить неофициальные `@types/yandex-maps`

Также типизация будет идти "из коробки" в Картах 2.0 (https://github.com/PNKBizz/vue-yandex-maps/discussions/60)
